go linter fails in travis ci

Run pushd go-controller
~/work/ovn-kubernetes/ovn-kubernetes/go-controller ~/work/ovn-kubernetes/ovn-kubernetes
curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b /home/runner/go/bin
golangci/golangci-lint info checking GitHub for latest tag
golangci/golangci-lint info found version: 1.24.0 for v1.24.0/linux/amd64
golangci/golangci-lint info installed /home/runner/go/bin/golangci-lint
level=error msg="Timeout exceeded: try increasing it by passing --timeout option"
make: *** [lint] Error 4
##[error]Makefile:45: recipe for target 'lint' failed
##[error]Process completed with exit code 2.

--- a/go-controller/hack/lint.sh
+++ b/go-controller/hack/lint.sh
@@ -1,6 +1,5 @@
 #!/usr/bin/env bash
-
 GO111MODULE=on ${GOPATH}/bin/golangci-lint run \
     --tests=false --enable gofmt \
-    --timeout=10m0s \
+    --timeout=15m0s --verbose --print-resources-usage --modules-download-mode=vendor \
     && echo "lint OK!"

